example/declarative.dart

```diff
@@ -14,14 +14,14 @@ init(GrinderContext context) {
 }
 
 @Task(
-    depends: const ['init'],
+    depends: const [init],
```

Comment:
In another PR, we may want to add to our declarative example (or in the readme) and example of a camelCase method name and the string name to refer to it by on the CLI.